tests/ssl-opt.sh

```diff
@@ -285,9 +286,8 @@ get_config_value_or_default() {
 requires_config_value_at_least() {
     VAL="$( get_config_value_or_default "$1" )"
     if [ -z "$VAL" ]; then
-        # Should never happen
-        echo "Mbed TLS configuration $1 is not defined"
-        exit 1
+        # when test-ref-config, MBEDTLS_SSL_MAX_EARLY_DATA_SIZE report undefine.
+        SKIP_NEXT="YES"
```

Comment:
Should we consider add "MBEDTLS_SSL_MAX_EARLY_DATA_SIZE" to `requires_all_configs_enabled` instead of change the behavior here?

Reply:
The issue is reported by shell script.
That won't resolve the script issue.  The sentence will be  called always.